redbot/cogs/streams/streams.py

```diff
@@ -38,6 +38,15 @@
 
 @cog_i18n(_)
 class Streams(commands.Cog):
+    """Various commands relating to streaming platforms.
+
+    You can check if a Twitch, YouTube, Picarto or Mixer stream is
+    currently live. Stream alerts can also be configured to
+    automatically post in a channel when a streamer goes live.
+
+    Twitch and YouTube require access to their API, so you need to
```

Comment:
I'm not doing a full review of this but I think that considering this help can be seen by any user, this line just shouldn't be here.